docs/DeveloperGuide.md

```diff
@@ -725,6 +772,22 @@ Also, we understand that some users may not wish to type leading zeros for days
 
 ### **9. Allow Multiple Indexes Input for Delete Command**
 
+**Feature Flaw in Current Implementation**
+
+Currently, users can only delete one entry at once. However, there will be cases where the user wish to delete multiple entries at once, especially if the user wishes to delete all the companies that he got rejected from. This can be tedious and inconvenient for the user.
```

Comment:
change entry to company

also standardise between users/the user?

Reply:
we are using the user?